modules/carto/src/style/utils.js

```diff
@@ -0,0 +1,80 @@
+import * as cartoColors from 'cartocolor';
```

Comment:
Let's rename this file to palette

Reply:
Great, and, much better if we create a new folders called `utils`? So => `style/utils/palette.js`